pkg/dpll-netlink/dpll.go

```diff
@@ -141,7 +142,7 @@ func ParseDeviceReplies(msgs []genetlink.Message) ([]*DoDeviceGetReply, error) {
 			case DPLL_A_MODULE_NAME:
 				reply.ModuleName = ad.String()
 			case DPLL_A_MODE:
-				reply.Mode = ad.Uint32()
+				//reply.Mode = ad.Uint32()
```

Comment:
We don't seem to use this mode anywhere

Reply:
we should remove this case; in case  if it exists and not used, we fall to default case ?